packages/react-next/src/components/OverflowSet/OverflowSet.test.tsx

```diff
@@ -69,6 +70,11 @@ describe('OverflowSet', () => {
     });
   });
 
+  isConformant({
```

Comment:
Probably rather than renaming OverflowSet.ts to tsx, the test should be updated to handle either ts or tsx.

Same with other components.